### PR TITLE
maintainers/teams: add golang

### PR DESCRIPTION
**Pull request by zowoq**
Easier for people to join and leave rather than being listed in each `go` package.

cc @cstrahan @Frostman @kalbasit @mic92 @orivej @rvolosatovs

### CODE REVIEW

**Review by rvolosatovs — APPROVED** [review]

**Review by Mic92 — APPROVED** [review]

**Review by kalbasit — APPROVED** [review]

**Comment by Mic92** [comment]
Maybe @c00w is interested too.

**Comment by c00w** [comment]
I'm definitely interested :)

**Review by c00w — APPROVED** [review]

File: maintainers/team-list.nix [hunk 1]
```diff
@@ -34,6 +34,19 @@ with lib.maintainers; {
...
 
+  golang = {
+    members = [
+      cstrahan
+      Frostman
+      kalbasit
+      mic92
+      orivej
+      rvolosatovs
+      zowoq
```
Comment (Mic92) [inline maintainers/team-list.nix:45]: Could you add @c00w here?

Comment (zowoq) [reply maintainers/team-list.nix:46]: Done.

**Review by Frostman — APPROVED** [review]

**Review by orivej — APPROVED** [review]

**Comment by kalbasit** [comment]
Seems like everyone's involved approved it.

**Pull request merged**